.gitignore

```diff
@@ -0,0 +1,24 @@
+### Haskell ###
+dist
+dist-*
+cabal-dev
+*.o
+*.hi
+*.chi
+*.chs.h
+*.dyn_o
+*.dyn_hi
+.hpc
+.hsenv
+.cabal-sandbox/
+cabal.sandbox.config
+*.prof
+*.aux
+*.hp
+*.eventlog
+.stack-work/
+cabal.project.local
+cabal.project.local~
+.HTF/
+.ghc.environment.*
+.ghci
```

Comment:
`.ghci` was not in the gitignore.io template, but in the .gitignore from the old wallet. It seemed useful, so I added it here too.